cli_client/python/timesketch_cli_client/commands/events.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2021 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Commands for events."""
+
+import sys
+import click
+import yaml
+
+
+@click.group("events")
+def events_group():
+    """Manage event."""
+
+
+@events_group.command("annotate")
+@click.option("--timeline-id", type=int, required=True)
+@click.option("--event-id", required=True, help="ID of the event.")
+@click.option(
+    "--tags",
+    required=False,
+    help="Comma seperated list of Tags to add to the event.",
+)
+@click.option("--comments", required=False, help="Comment to add to the event.")
+@click.option(
+    "--output-format",
+    "output",
+    required=False,
+    help="Set output format (overrides global setting)",
+)
+@click.pass_context
+def annotate(ctx, timeline_id, event_id, tags, comments, output):
```

Comment:
I would opt to make these singular:
`--tag` instead of `--tags`

Reply:
I `--tag` an event with one or more tags
I `--comment` on an event with comment

Reply:
ack